live/terragrunt.hcl

```diff
@@ -1,17 +1,16 @@
 locals {
   cluster_name = "mqperf-cluster"
-  region       = "eu-central-1"
 }
 
 
 remote_state {
   backend = "s3"
   config = {
     encrypt        = true
-    bucket         = "ab-new-bucket"
+    bucket         = TF_VAR_BUCKET_NAME
```

Comment:
I think we need to use built-in function `get_env(NAME)` here, like this `get_env("TF_VAR_BUCKET_NAME")`